code/datums/jobs/job/shipside.dm

```diff
@@ -404,9 +406,9 @@ If you are not piloting, there is an autopilot fallback for command, but don't l
 	switch(playtime_mins)
 		if(0 to 1500) // starting
 			new_human.wear_id.paygrade = "O2"
-		if(1501 to 6000) // 25 hrs
+		if(1501 to 3000) // 25 hrs
 			new_human.wear_id.paygrade = "O3"
-		if(6001 to INFINITY) // 50 hrs
+		if(3001 to INFINITY) // 50 hrs
 			new_human.wear_id.paygrade = "O4"
```

Comment:
I said SO and CMO changes only

Reply:
As said in the PR desc, this is a bug fix. The listed time in the comment is 50 hrs, the actual time is 100 hours. I simply fix it to match the listed/intended time of 50 hrs. 

If you want I can change the comment to 100 hrs and keep the current time, but that wouldn't match the current trend of three rank roles at all and seems kind of excessive. Regardless I'd like to do something about this, because this is not intended regardless of how you look at it.